levels/tilesets/sand_tileset.tsx

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please do not add tilesets to the repository. after creating the ship map i think that it will be better if the tilesets are not part of the repository because they will probably differ and change much during the development and there is no point in tracking all that in the repository

Reply:
k